Src/LiquidProjections.PollingEventStore/Subscription.cs

```diff
@@ -91,7 +92,7 @@ public void Start()
                 Page page = await TryGetNextPage(lastProcessedCheckpoint).ConfigureAwait(false);
                 if (page != null)
```

Comment:
🤡 If the TryGetNextPage would return an empty page instead of null, the code would be simpler.

Reply:
I tried that, but I need to be able to differentiate between an empty page and a failed attempt.